app/index.tsx

```diff
@@ -204,8 +206,65 @@ export default function Index({ children }: RootLayoutProps) {
     fetchBalances(true)
     fetchFeesList()
     fetchStakingDelegations()
+    fetchPrices()
   }, [isConnected, address])
 
+  const fetchPrices = useCallback(
+    debounce(async () => {
```

Comment:
Why is it needed to debounce this? It looks like it's only executed on the above effect that triggers with `isConnected` and `address`

Reply:
It's not strictly needed here, just a precaution in case this function is called in another component. Or in case a user clicks a refresh button multiple times—it's not going be doing unnecessary requests.